src/Sylius/Bundle/AdminBundle/Notification/CompositeNotificationProvider.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\Notification;
+
+use Traversable;
```

Comment:
According to symfony CS, root level classes should be used directly as `\Traversable`